docs/csharp/quick-starts/list-collection.yml

```diff
@@ -0,0 +1,167 @@
+  audience: Developer
+  level: Beginner
+  displayType: two-column
+  interactive: csharp
+items:
+- durationInMinutes: 1
+  content: |
+    This tutorial teaches you C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that create, modify and explore collections and arrays.
+- title: Create lists
+  durationInMinutes: 2
+  content: |
+    Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the **Run** button:
+
+    ```csharp
+    var names = new List<string> { "<name>", "Ana", "Felipe" };
+    foreach (var name in names)
+    {
+      Console.WriteLine($"Hello {name.ToUpper()}!");
+    }
+    ```
+
+    You've just created a list of strings, added three names to that list, and printed out the names in all CAPS. You're using concepts
+    that you've learned in earlier quick starts to loop through the list.
+
+    The code to display names makes use of **interpolated strings**.  When you precede a `string` with the `$` character, you can embed C# code in the string declaration. The actual string replaces that C# code with the value it generates. In this example, it replaces the `{name.ToUpper()}` with each name, converted to capital letters, because you called the <xref:System.String.ToUpper%2A> method.
+
+    Let's keep exploring.
+    
+    > [!NOTE]
+    > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
+
+- title: Modify list contents
+  durationInMinutes: 3
+  content: |
+    The collection you created using the <xref:System.Collections.Generic.List%601> type. This type stores sequences of elements. You specify the type of the elements between the angle brackets.
+    
+    One important aspect of this <xref:System.Collections.Generic.List%601> type is that it can grow or shrink, enabling you to add or remove elements. Add this code below the code you've already written:
+
+    ```csharp
+    Console.WriteLine();
+    names.Add("Maria");
+    names.Add("Bill");
+    names.Remove("Ana");
+    foreach (var name in names)
+    {
+      Console.WriteLine($"Hello {name.ToUpper()}!");
+    }
+    ```
+
+    You've added two more names to the end of the list. You've also removed one as well.
+    
+    The <xref:System.Collections.Generic.List%601> enables you to reference individual items by **index** as well. You access items using the `[` and `]` tokens. Add this code to the bottom and try it:
+
+    ```csharp
+    Console.WriteLine($"My name is {names[0]}");
+    Console.WriteLine($"I've added {names[2]} and {names[3]} to the list");
+    ```
+
+    You're not allowed to access past the end of the list. You can check how long the list is using the `<xref:System.Collections.Generic.List%601.Count%2A>` property:
```

Comment:
the link is styled as code. it probably shouldn't otherwise it looks like a broken link

also, do you want to be super direct and tell them to add this code too?